spec/dummy/app/assets/config/manifest.js

```diff
@@ -0,0 +1,3 @@
+//= link_tree ../images
+//= link_directory ../javascripts .js
+//= link_directory ../stylesheets .css
```

Comment:
Expected space or tab after '//' in comment      spaced-comment<br>Newline required at end of file but not found    eol-last